tests/frame/schema_test.py

```diff
@@ -222,7 +221,7 @@ def test_nested_dtypes() -> None:
     nwdf = nw.from_native(df)
     assert nwdf.schema == {"a": nw.List, "b": nw.Array(nw.Int64, 2), "c": nw.Struct}
     df = pl.DataFrame(
-        {"a": [[1, 2]], "b": [[1, 2]], "c": [{"a": 1}]},
+        {"a": [[1, 2]], "b": [[1, 2]], "c": [{"a": 1, "b": "x", "c": 1.1}]},
```

Comment:
at the end of this test, there's
```python
assert nwdf.schema == {"a": nw.List, "b": nw.Array(nw.Int64, 2), "c": nw.Struct}
```
maybe we should make it
```python
assert nwdf.schema == {"a": nw.List, "b": nw.Array(nw.Int64, 2), "c": nw.Struct}
assert nwdf.schema == {"a": nw.List(nw.Int64), "b": nw.Array(nw.Int64, 2), "c": nw.Struct({'a': nw.Int64, 'b': nw.String, 'c': nw.Float64}}
```
? and similarly for the others here in this file, just to double-check that we're really covering everything (and doing so across the different backends)

Reply:
Done, thanks for the suggestion.